frontend/src/components/AdminModals/AdminSuspendModal.js

```diff
@@ -27,22 +27,25 @@ function AdminSuspendModal(props) {
     </button>
   );
 
+  const toggleSuspendText = (isSuspended) => {
+    const textForSuspendedUser = 'Clicking "UnSuspend" will allow the user to logging into the app. The user will remain unsuspended until this status is removed by an administrator.If you do not wish to unsuspend the user, close this dialog box.';
+    const textForActiveUser = 'Clicking "Suspend" will prevent the user from logging into the app. The user will remain suspended until this status is removed by an administrator.If you do not wish to suspend the user, close this dialog box.';
+    if (isSuspended) return textForSuspendedUser;
+    return textForActiveUser;
```

Comment:
I think you can use a ternary here, like `return isSuspended ? textforSuspended : textForActive`.